dart/collision/fcl/FCLTypes.hpp

```diff
@@ -55,25 +79,97 @@ template <class T> using fcl_shared_ptr = boost::shared_ptr<T>;
 template <class T> using fcl_weak_ptr = boost::weak_ptr<T>;
 #endif
 
-namespace dart {
-namespace collision {
+namespace dart
+{
+namespace fcl
+{
+#if FCL_VERSION_AT_LEAST(0,6,0)
+// Geometric fundamentals
+typedef ::fcl::Vector3<double> Vector3;
```

Comment:
I saw that that was the existing usage, but wasn't sure how to resolve conflicts like dart::collision::FCLCollisionObject, which is already a Dart class.